charts/tfy-karpenter-config/values.yaml

```diff
@@ -23,7 +23,7 @@ karpenter:
     ## @param karpenter.defaultNodeTemplate.extraTags [object] Additional tags for the node template.
     extraTags: {}
     ## @param karpenter.defaultNodeTemplate.amiFamily AMI family to use for node template
-    amiFamily: ""
+    amiFamily: "AL2023"
```

Comment:
Please don't apply this, When karpenter was migrated to v1 the webhook migrated itself to use `amiSelectorTerms` and not `amiFamily` so if you apply this a drift will be detected in the nodetemplate causing all nodes to disrupt